app/models/activity_insight_oa_status_exporter.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class ActivityInsightOAStatusExporter < ActivityInsightExporter
+  def initialize(files, target)
+    super()
+    @files = files
+    # target should be 'beta' or 'production'
+    @target = target
+  end
+
+  def export
+    logger = Logger.new('log/ai_oa_status_export.log')
+    logger.info "Open access status export to #{target} Activity Insight started at #{DateTime.now}"
+    not_exported_ids = []
+    files.each do |file|
+      response = HTTParty.post webservice_url, body: to_xml(file),
+                                               headers: { 'Content-type' => 'text/xml' }, basic_auth: auth, timeout: 180
+      if response.code != 200
+        logger.error Nokogiri::XML(response.to_s).text
+        logger.error "File ID: #{file.id}"
+        not_exported_ids << file.id
+      elsif target == 'production'
+        publication = file.publication
+        publication.exported_oa_status_to_activity_insight = true
+        publication.save!
+      end
+    end
+
+    logger.info "Open access status export to #{target} Activity Insight ended at #{DateTime.now}"
+    logger.info "Files not exported to AI: #{not_exported_ids}"
+  end
+
+  private
+
+    attr_accessor :files, :target
+
+    def to_xml(file)
+      builder = Nokogiri::XML::Builder.new(encoding: 'UTF-8') do |xml|
+        xml.Data do
+          user = file.user
+          xml.Record('username' => user.webaccess_id) do
+            xml.INTELLCONT('id' => file.intellcont_id) do
+              xml.POST_FILE('id' => file.post_file_id) do
+                xml.ACCESIBLE('Already Openly Available')
+              end
+              xml.RMD_ID(file.publication.id, access: 'READ_ONLY')
```

Comment:
I know it says to update the `RMD_ID` in the issue, and I _think_ we can keep this, but we'll have to consider that once this value is set these records stop importing from Activity Insight.  I think this is probably okay since it's open access, but we might want to double check this with Ana or Nicole.

Reply:
I know we discussed this, but just to document- Ana said let's update `RMD_ID` for now, and we can reassess down the road.